packages/availity-react-kit/project/app/App.js

```diff
@@ -1,6 +1,31 @@
-import React from 'react';
-import Request from './request/Request';
+import React, { Component } from 'react';
+import { BrowserRouter, Route } from 'react-router-dom';
+import AuthorizationsRequest from './Request';
+import AuthorizationsResponse from './Response';
 
-const App = () => <Request />;
+class App extends Component {
+  constructor() {
+    super();
+
+    this.renderRootPath = ::this.renderRootPath;
+  }
+
+  renderRootPath() {
```

Comment:
You can avoid the constructor boilerplate needed for binding by using an arrow function here
```
renderRootPath = () => {
```
then remove Lines 7-12

Reply:
i'll update